packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/languages/talon/clearEveryValue2.yml

```diff
@@ -0,0 +1,32 @@
+languageId: talon
+command:
+  version: 5
+  spokenForm: clear every value
+  action: {name: clearAndSetSelection}
+  targets:
+    - type: primitive
+      modifiers:
+        - type: everyScope
+          scopeType: {type: value}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: |-
+    tag: user.aaa
+    tag: user.bbb
+    -
+    aaa: bbb
+  selections:
+    - anchor: {line: 0, character: 0}
+      active: {line: 0, character: 0}
+  marks: {}
+finalState:
+  documentContents: |-
+    tag:
+    tag:
```

Comment:
These lines should have spaces at the end; see https://github.com/wenkokke/tree-sitter-talon/issues/22

Reply:
PR created
https://github.com/wenkokke/tree-sitter-talon/pull/23